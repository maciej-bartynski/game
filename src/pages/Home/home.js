import React from 'react';
import { View, Text } from 'react-native';
import ComponentExample from 'components/Component-example';

const style = { flex: 1, justifyContent: 'center', alignItems: 'center' };
export default function Home() {
  return (
    <View style={style}>
      <Text>Home page</Text>
      <ComponentExample label="Button was clicked times" />
    </View>
  );
}
