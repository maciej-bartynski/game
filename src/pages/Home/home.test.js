import React from 'react';
import renderer from 'react-test-renderer';
import WrapWithRedux from 'app/wrapWithRedux';
import Home from './index';

describe('<Home />', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<WrapWithRedux><Home /></WrapWithRedux>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
