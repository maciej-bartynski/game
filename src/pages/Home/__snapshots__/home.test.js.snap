// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<Home /> matches snapshot 1'] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Home page
  </Text>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderRadius": 5,
        "borderWidth": 1,
        "opacity": 1,
        "padding": 10,
      }
    }
    testID="componentExample__button"
  >
    <Text
      testID="componentExample__text"
    >
      Button was clicked times
       
      :
       
      0
    </Text>
  </View>
</View>
`;
